---
geometry_msgs/TransformStamped device
float32[] plane_coefficients